pkgs/applications/file-managers/yazi/default.nix

```diff
@@ -60,6 +61,10 @@ rustPlatform.buildRustPackage rec {
     ''
       wrapProgram $out/bin/yazi \
          --prefix PATH : "${makeBinPath runtimePaths}"
+      installShellCompletion --cmd ${pname} \
```

Comment:
```suggestion
      installShellCompletion --cmd yazi \
```

Reply:
If we add this contribution, we need to change also https://github.com/XYenon/nur-packages/commit/2c201c14ded01af4f7325d5cdaf14b7ffe9f232d @XYenon.

Reply:
> If we add this contribution, we need to change also [XYenon/nur-packages@2c201c1](https://github.com/XYenon/nur-packages/commit/2c201c14ded01af4f7325d5cdaf14b7ffe9f232d) @XYenon.

Yeah, I have updated my nur

Reply:
is the nur needed now there is a flake \o/ ?